agenta-web/src/components/Playground/Views/ParametersCards.tsx

```diff
@@ -1,8 +1,106 @@
 import {Row, Card, Slider, Select, InputNumber, Col, Input, Button} from "antd"
-import React, {ParamHTMLAttributes} from "react"
+import React from "react"
 import {Parameter, InputParameter} from "@/lib/Types"
 import {renameVariables} from "@/lib/helpers/utils"
-import {useEffect} from "react"
+import {createUseStyles} from "react-jss"
+
+const useStylesModalParams = createUseStyles({
+    row1: {
+        padding: "0px 0px",
+        width: "100%",
+        marginTop: "16px",
+    },
+    card: {
+        marginTop: 16,
+        width: "100%",
+        border: "1px solid #ccc",
+        marginRight: "24px",
+        "& .ant-card-body": {
+            padding: "4px 16px",
+            margin: "16px 0px",
+            border: "0px solid #ccc",
+        },
+        "& .ant-card-head": {
+            minHeight: 44,
+            padding: "0px 12px",
+        },
+    },
+    row2: {
+        alignItems: "center",
+        marginBottom: 10,
+    },
+    colTitle: {
+        margin: 0,
+        padding: 0,
+        textAlign: "left",
+    },
+    colSlider: {
+        marginBottom: 8,
+    },
+    inputNumber: {
+        margin: "0 16px",
+        width: "100%",
+    },
+    select: {
+        width: "100%",
+    },
+})
+const useStylesStringParams = createUseStyles({
```

Comment:
Instead of creating separate useStyles hooks for each component here, I would create only one and use it in all 3 components. That way, a lot of styles could be reused like row, card